packages/vite-plugin-svelte/src/utils/options.ts

```diff
@@ -207,10 +207,7 @@ export function resolveOptions(
 	enforceOptionsForProduction(merged);
 	// mergeConfigs would mangle functions on the stats class, so do this afterwards
 	const isLogLevelInfo = [undefined, 'info'].includes(viteConfig.logLevel);
-	const disableCompileStats = merged.experimental?.disableCompileStats;
-	const statsEnabled =
-		disableCompileStats !== true && disableCompileStats !== (merged.isBuild ? 'build' : 'dev');
-	if (statsEnabled && isLogLevelInfo) {
+	if (isLogLevelInfo) {
```

Comment:
removing the statsEnabled flag here would cause the stat collection to always happen unless user sets loglevel to warning or above, which would be out of sync with the change to loglevel debug for stat output.

I'm still not sure why moving the logging to a debug namespace is better than flipping the default value of the stats enabled option.

Reply:
Updated to check based on `log.debug.enabled` (https://github.com/sveltejs/vite-plugin-svelte/pull/614/commits/59a1034bfe681c8da10b31185d63fb4f65bfe078).

I guess for me the lesser the options the better. And logging stats is kind of a debug info that should be tucked with other debug logs.